pkg/query-service/app/http_handler.go

```diff
@@ -337,12 +337,7 @@ func (aH *APIHandler) updateDashboard(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	if postData["uuid"] != uuid {
```

Comment:
I think we need this check to make sure the _update_ operation on _**A**_ dashboard doesn't overrides the UUID and other data of the same to _**B**_ dashboard.

Moreover, in case of update operation `uuid` _must be present_